configuration/pih/htmlforms/hiv/section-hiv-plan_v3.0.xml

```diff
@@ -671,83 +671,19 @@
-                                AZT/3TC 60/30 mg,
-                                TDF/3TC 300/300 mg,
-                                ABC/3TC 300/300 mg,
-                                AZT/3TC/NVP 60/30/50 mg,
-                                RAL/ETV 400/100 mg,
-                                DTG 50 mg,
-                                AZT 10 mg sp,
-                                NVP 10 mg sp,
-                                NVP 50 mg,
-                                TDF/3TC/EFV 300/300/600 mg,
-                                3TC 10 mg sp,
-                                ABC/3TC 60/30 mg,
-                                RAL/DRV/r 400/600/100mg,
-                                DRV/ETV 300/100 mg,
-                                ABC/3TC 120/60 mg" />
-
-                        <label>ARV #2</label>
-                        <obs id="arv2Drug" conceptId="PIH:13162"
-                             answerDrugIds="
-                                78fab02a-dfbe-11e9-8a34-2a2ae2dbcce4,
-                                78f96210-dfbe-11e9-8a34-2a2ae2dbcce4,
-                                78faa576-dfbe-11e9-8a34-2a2ae2dbcce4,
-                                78f9780e-dfbe-11e9-8a34-2a2ae2dbcce4,
-                                78f96526-dfbe-11e9-8a34-2a2ae2dbcce4,
-                                78f981d2-dfbe-11e9-8a34-2a2ae2dbcce4,
-                                78faaaf8-dfbe-11e9-8a34-2a2ae2dbcce4,
-                                35c6041e-0af3-4bab-887d-9db682a02248,
-                                78f976c4-dfbe-11e9-8a34-2a2ae2dbcce4,
-                                78faa710-dfbe-11e9-8a34-2a2ae2dbcce4,
-                                78f98308-dfbe-11e9-8a34-2a2ae2dbcce4,
-                                78f95e78-dfbe-11e9-8a34-2a2ae2dbcce4,
-                                78f95d38-dfbe-11e9-8a34-2a2ae2dbcce4,
-                                78f97b9c-dfbe-11e9-8a34-2a2ae2dbcce4,
-                                29c9dd27-25f2-45c6-9708-93fe142a46ba,
-                                0fc3d5c1-fd39-4899-b5c0-b094e28ff359"
-                             answerLabels="
-                                DTG 50 mg,
-                                EFV 600 mg,
-                                TDF 300 mg,
-                                NVP 200 mg,
-                                NVP 50 mg,
-                                ABC 300 mg,
-                                LPV/r 40/10 mg,
-                                ABC 60 mg,
-                                LPV/r 100/25 mg,
-                                LPV/r 80/20 mg,
-                                ABC 20 mg,
-                                LPV/r 200/50 mg,
-                                ATV/r 300/100 mg,
-                                EFV 200 mg,
-                                ETV 100mg,
-                                DRV/ETV 300/100 mg" />
+                             answerConceptIds="3cd43a64-26fe-102b-80cb-0017a47871b2,
```

Comment:
This form is now only being used for old, migrated data, and the only data we are migrating in are the concepts listed below.